lib/msal-core/src/Constants.ts

```diff
@@ -80,6 +80,9 @@ export class Constants {
 
   static get cacheLocationLocal(): CacheLocation { return "localStorage"; }
   static get cacheLocationSession(): CacheLocation { return "sessionStorage"; }
+
+  static get interactionTypeRedirect(): InteractionType { return "redirectInteraction"; }
+  static get interactionTypePopup(): InteractionType { return "popupInteraction"; }
```

Comment:
are there other places this could go? This file is already so bloated

Reply:
I tried to keep it organized, we should probably have a separate PR to refactor the Constants file. I am open to suggestions.